languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10026,6 +10026,35 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_OU_QU_ELLE" name="quelle ou qu'elle" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="SENT_START"></token>
+                  <marker>
+                      <token regexp="yes">qu(&apostrophe;)?</token>
+                      <token regexp="yes">elle</token>
+                  </marker>
+                  <token postag="V.*" postag_regexp="yes" skip="-1"><exception postag="V.* sub|V avoir sub.*|V etre sub.*" postag_regexp="yes"></exception></token>
+                  <token postag="M fin inte"></token>
+              </pattern>
+              <message>"Qu'elle" est une erreur grammaticale dans une phrase interrogative. Souhaitez-vous écrire <suggestion>Quelle</suggestion> ?</message>
```

Comment:
See my comment in QUELLES: We should not make such broad statements like "this word is a grammatical error in a question". What's more, this is not really a _grammatical_ error.